### PR TITLE
Add a target for running edpm-ansible playbooks

### PR DESCRIPTION
**Pull request by fao89**
As a quick way to debug, you can build the edpm-ansible image locally and run 
```
git clone https://github.com/openstack-k8s-operators/edpm-ansible.git
cd edpm-ansible
touch playbooks/my_new_playbook.yaml
make openstack_ansibleee_build

cd ~/install_yamls
DATAPLANE_PLAYBOOK=osp.edpm.my_new_playbook make edpm_ansible_runner
```

### CODE REVIEW

File: devsetup/scripts/edpm-ansible-runner.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+pushd ${SCRIPTPATH}
+pushd ${OUTPUT_DIR}
+
+oc get secret ${EDPM_INVENTORY_SECRET} -n ${NAMESPACE} -o json | jq '.data | map_values(@base64d)' | jq -r .inventory > inventory
+podman run --rm -ti \
+            -e "ANSIBLE_ENABLE_TASK_DEBUGGER=true" \
+            -e "ANSIBLE_FORCE_COLOR=true" \
+            -e "ANSIBLE_SSH_ARGS=-C -o ControlMaster=auto -o ControlPersist=80s" \
+            --volume "$(pwd)/inventory":/runner/inventory/hosts:Z \
+            --volume "$(pwd)/ansibleee-ssh-key-id_rsa":/runner/env/ssh_key:Z \
```
Comment (fao89) [inline devsetup/scripts/edpm-ansible-runner.sh:35]: maybe we can share a volume with the local edpm-ansible?

**Review by fao89 — COMMENTED** [review]
I'm still wondering if the script should build the local image first, or if we let the user do it

**Comment by fao89** [comment]
recheck

File: devsetup/scripts/edpm-ansible-runner.sh (hunk 1 above)
Comment (jistr) [reply devsetup/scripts/edpm-ansible-runner.sh:35]: Yea or add a generic ${OPENSTACK_RUNNER_ARGS} where folks can set their own extra arguments, including additional `--volume` (as long as the path doesn't contain a space...).

**Review by jistr — APPROVED** [review]

**Pull request merged**